### PR TITLE
[risk=low][RW-7256] remove Registration Dashboard and other DAR=false code

### PR DESCRIPTION
**Pull request by jmthibault79**
Covers steps 2-4 of https://precisionmedicineinitiative.atlassian.net/browse/RW-7256

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/homepage/homepage.tsx [hunk 1]
```diff
@@ -212,74 +186,15 @@ export const Homepage = fp.flow(withUserProfile(), withNavigation, withRouter)(c
     clearTimeout(this.timer);
   }
 
-  // This method is part of the eRA Commons linkage flow. The "return-url" sent to the Shibboleth
-  // server is set to "nih-callback", which is configured in our router to point to the homepage
-  // component. If we reach this component with a 'token' query parameter, we will attempt to
-  // store that token in Terra.
-  async validateNihToken() {
```
Comment (jmthibault79) [inline ui/src/app/pages/homepage/homepage.tsx:219]: these are handled by data-access-requirements now

File: ui/src/app/pages/homepage/homepage.tsx [hunk 2]
```diff
@@ -212,74 +186,15 @@ export const Homepage = fp.flow(withUserProfile(), withNavigation, withRouter)(c
...
   setFirstVisit() {
     this.setState({firstVisit: true});
     profileApi().updatePageVisits({ page: this.pageId});
   }
 
-  getRegistrationTasksMap() {
-    return getRegistrationTasksMap(this.props.navigate);
-  }
-
+  // TODO reload profile?
```
Comment (als364) [inline ui/src/app/pages/homepage/homepage.tsx:194]: Can you please create a ticket for each TODO and put the JIRA ticket number in the comment?

File: ui/src/app/pages/homepage/homepage.tsx [hunk 3]
```diff
@@ -302,34 +217,16 @@ export const Homepage = fp.flow(withUserProfile(), withNavigation, withRouter)(c
...
-            (() => !!(this.getRegistrationTasksMap()['rasLoginGov']
-              .completionTimestamp(profile)))() : true),
-        dataUserCodeOfConductCompleted:
-          (() => !!(this.getRegistrationTasksMap()['dataUserCodeOfConduct']
-            .completionTimestamp(profile)))()
-      });
-      // TODO(RW-6493): Update rasCommonsLinked similar to what we are doing for eraCommons
 
-      const workbenchAccessTasks = parseQueryParams(this.props.location.search).get('workbenchAccessTasks');
+      // TODO keep this check?
```
Comment (als364) [inline ui/src/app/pages/homepage/homepage.tsx:221]: ditto JIRA

**Review by als364 — APPROVED** [review]
This is great, simplifies the code a lot. just some nitpicks about creating tickets to TODOs.

**Pull request merged**